src/Hangman/Domain/Repositories/GamesRepository.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { Game } from '../AggregateRoot/Game.aggregate';
+
+@Injectable()
+export class GamesRepository {
+  // repository heeft niks in zich, geen state. alleen het taakje 'maak nieuw aggregate, set data, start game (laatste stap is apply event op aggregate)
```

Comment:
@weemen wat voor nut heeft de repository hier? Dit stapje kan ook in de command handler gedaan worden?

Reply:
Dat ga je vanzelf zien als je guessLetter gaat implementeren. 

Een repository is eigenlijk een mooie interface om objecten (in dit geval aggregates)  op te lagen zonder dat jij je druk hoeft te maken welke storage systeem hierachter hangt.

Vaak doet de repository ook de opslag van de aggregate maar misschien dat nestjs net ff iets anders werkt. Dat gaan we vanzelf zien denk ik.

Vaak vind je hier methods als:
save()
findOne()
findAll()

Reply:
aha :) top, thanks!